src/pystatis/table.py

```diff
@@ -162,10 +177,22 @@ def parse_zensus_table(data: pd.DataFrame) -> pd.DataFrame:
         time_label = data["time_label"].iloc[0]
         time = pd.DataFrame({time_label: pivot_table["time"]})
 
+        ags_code = None
+        pos_of_ags_col = np.where(data.iloc[0].isin(config.ZENSUS_AGS_CODES))[0]
+        if pos_of_ags_col.size > 0:
+            pos_of_ags_col = pos_of_ags_col[0]
+            label = "Amtlicher Regionalschlüssel (ARS)"  # en: official municipality code (AGS)
```

Comment:
Will this cause issues once language = 'en' is provided?

Reply:
Yes but this ist fine for now as this PR is isolated from your feature. Whoever is first can just merge in dev and then we have to take care of proper translation but we can check once we have the language PR merged

Reply:
But as you can see I already had en in mind :)